code/modules/cargo/exports/weapons.dm

```diff
@@ -69,4 +69,37 @@
 	cost = 3
 	unit_name = "pair"
 	message = "of handcuffs"
-	export_types = list(/obj/item/restraints/handcuffs)
\ No newline at end of file
+	export_types = list(/obj/item/restraints/handcuffs)
+	
+// relics of lavaland
+
+/datum/export/weapon/hierophant
+	cost = 50000
+	unit_name = "Hierophant Club"
+	export_types = list(/obj/item/hierophant_club)
+	
+/datum/export/weapon/lava
+	cost = 45000
+	unit_name = "Lava Staff"
+	export_types = list(/obj/item/lava_staff)
+	
+/datum/export/weapon/cleaving_saw
+	cost = 40000
+	unit_name = "Cleaving Saw"
+	export_types = list(/obj/item/melee/transforming/cleaving_saw)
+	
+/datum/export/weapon/immortality_talisman
+	cost = 25000
+	unit_name = "Immortality Talisman"
+	export_types = list(/obj/item/device/immortality_talisman)
+	
+/datum/export/weapon/mayhem
+	cost = -5000
```

Comment:
If the point is to add uses to things that are normally for antagonists only, then what is the point of this one? -5000. Griefing, maybe?

Reply:
well you bring a terrible thing to centcom that causes chaos at their cargo

Reply:
All of those things can cause chaos at their cargo if used inappropriately so that's kind of a moot point.

Reply:
If it doesn't trigger something fun in a cargo event then its probably not worth the time.

Reply:
Why do you lose money from a boss item when the rest give you over 25k?

Reply:
Just don't do this, you can set the price to 0